### PR TITLE
Modify ConditionalGradient optimizer with handling case of gradient be 0

### PR DESCRIPTION
**Pull request by pkan2**
Hello!

I have modified the ConditionalGradient optimizer for handling the case of gradient be 0. I add in a constant 'epsilon' into the formula expression, which is relatively tiny as compared to frobenius norm of gradient.

Therefore, frobenius norm of gradient will be dominant, when gradient is not zero. But 'epsilon' will be dominant, when the frobenius norm of gradient is zero. 

In this way, it avoids the case of un-defined operation to be divided by zero. 

Thanks!

### CODE REVIEW

**Comment by pkan2** [comment]
Hi @WindQAQ ! Thank you so much for your feedbacks! 

I have finished modifying based on your review and the link you provided! Thanks!

Sincerely, 

Pengyu Kan

File: tensorflow_addons/optimizers/conditional_gradient.py [hunk 1]
```diff
@@ -50,19 +56,23 @@ def __init__(self,
...
+            epsilon: A `Tensor` or a floating point value. A small constant for numerical
+                    stability when handling the case of norm of gradient to be zero.
             use_locking: If `True` use locks for update operations.
             name: Optional name prefix for the operations created when
                 applying gradients.  Defaults to 'ConditionalGradient'
         """
         super(ConditionalGradient, self).__init__(name=name, **kwargs)
         self._set_hyper('learning_rate', kwargs.get('lr', learning_rate))
         self._set_hyper('lambda_', lambda_)
+        self.epsilon = epsilon
```
[the remark this answers is not in the document]
Comment (pkan2) [reply tensorflow_addons/optimizers/conditional_gradient.py]: Hi! @WindQAQ Thanks for pointing it out! It is modified!

Thanks!
